components/prism-abap.js

```diff
@@ -0,0 +1,47 @@
+		lookbehind: true
+	},
+	/* Numbers can be only integers. Decimal or Hex appear only as strings */
+	'number' : /\b\d+\b/,
+	/* Operators must always be surrounded by whitespace, they cannot be put 
+	adjacent to operands. 
+	*/
+	'operator' : {
+		pattern: /(\s)(\+|-|\*{1,2}|\/|<|>|<=|>=|=|\?=|<>)(?=\s)/,
+		lookbehind: true
+	},
+	'string_operator' : {
+		pattern: /(\s)(&{1,2})(?=\s)/,
```

Comment:
The parentheses aren't needed around `&{1,2}`. Also, it would be shorter written as `&&?`. (BTW this also apply to `\*{1,2}` in the operator regexp above.)